packages/types/src/clerk.ts

```diff
@@ -329,6 +329,14 @@ export interface Clerk {
    */
   unmountOrganizationSwitcher: (targetNode: HTMLDivElement) => void;
 
+  /**
+   * Prefetches the data displayed by an organization switcher.
+   * It can be used when `mountOrganizationSwitcher({ asStandalone: true})`, to avoid unwanted loading states.
+   * @experimantal This API is still under active development and may change at any moment.
+   * @param props Optional user verification configuration parameters.
+   */
+  __experimental_prefetchOrganizationSwitcher: () => void;
```

Comment:
We'd still need a way to initiate this from `clerk-react`, since `mountOrganizationSwitcher` will mount it as standalone. This is because the SWR cache is different in the clerk-js bundle and the one from the host application.

Prefetching already happens today via Organization Switcher trigger. So we are not making additional requests even with this change.

Reply:
This is mostly for internal usage and i was thinking when this goes GA to rename to `__internal_prefetchOrganizationSwitcher`

Reply:
I wouldn't expect people to call this manually, since `asProvider` will do the work for them.